packages/certs-v5/lib/notify.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const path = require('path')
+
+module.exports = {
+  notify: function (subtitle, message, success=true) {
+    const contentImage = path.join(__dirname, `../assets/${success ? 'success' : 'error'}.png`)
```

Comment:
Where those assets come from? Are we licensed to use them? Sorry for being the party popper, I really enjoy images on CLI, but you know, lawyers

Reply:
I copied them from another command (addons-v5), I guess that if it's fine for them it's fine for us to use them